docs/pages/enroll-resources/database-access/enroll-self-hosted-databases/oracle-self-hosted.mdx

```diff
@@ -258,6 +258,47 @@ $ tsh db logout oracle
 $ tsh db logout
 ```
 
+## Troubleshooting
+
+
+### Error on client connection
+
+If you are getting an error similar to the one below, that is likely a failure to connect
+to the TCPS service from the Teleport Database Service.
+
+```code
+$ tsh db connect --db-name=example --db-user=alice oracle
+SQLcl: Release 24.4 Production on Fri Feb 28 13:00:42 2025
+
+Copyright (c) 1982, 2025, Oracle.  All rights reserved.
+
+Connection failed
+  USER          =
+  URL           = jdbc:oracle:thin:@localhost:52019/example
+  Error Message = ORA-17800: Got minus one from a read call. (CONNECTION_ID=1234==)
+https://docs.oracle.com/error-help/db/ora-17800/
+
+ERROR: exit status 1
+```
+
+To resolve first check that `tcps` port is listening with the expected certificate with `lsnrctl status`.
```

Comment:
```suggestion
To resolve this error, first check that the `tcps` port is listening with the expected certificate with `lsnrctl status`.
```

Also, what is a "tcps port"? What should the user expect to see in `lsnrctl status`? What actions should they take if they don't see the expected output?